code/modules/clothing/rogueclothes/hats.dm

```diff
@@ -576,8 +576,23 @@
 	armor_class = ARMOR_CLASS_MEDIUM
 	block2add = FOV_BEHIND
 
+/obj/item/clothing/head/roguetown/helmet/astratahelm/alt
+	desc = "Gilded gold and silvered metal, the bright, vibrant colors of an Astratan crusader radiate from this blessed helmet."
+	icon_state = "astratahelm_alt"
+	item_state = "astratahelm_alt"
+
+/obj/item/clothing/head/roguetown/helmet/heavy/malumhelm
+	name = "malummite helmet"
+	desc = "Forged in a coal-black, this helmet carries a sigiled blade upon it's visor, ever reminding it's wearer of Malum's powerful gaze."
+	icon_state = "malumhelm"
+	item_state = "malumhelm"
+	emote_environment = 3
+	flags_inv = HIDEEARS|HIDEFACE|HIDEHAIR
+	block2add = FOV_RIGHT|FOV_LEFT
+	smeltresult = /obj/item/ingot/steel
```

Comment:
I know im pulling a 🤓 ☝️  But smelt result is already defined in /obj/item/clothing/head/roguetown/helmet
as steel

![image](https://github.com/user-attachments/assets/38bbd539-b5ad-4482-b97c-f8ab71e8b8a0)

Reply:
armor_class is as well in helmet/heavy

![image](https://github.com/user-attachments/assets/a9497db6-7e8a-4e04-8078-4ea3a10579b7)

Reply:
> armor_class is as well in helmet/heavy
> 
> ![image](https://private-user-images.githubusercontent.com/94647521/382135846-a9497db6-7e8a-4e04-8078-4ea3a10579b7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NjUxNDUsIm5iZiI6MTczMDQ2NDg0NSwicGF0aCI6Ii85NDY0NzUyMS8zODIxMzU4NDYtYTk0OTdkYjYtN2U4YS00ZTA0LTgwNzgtNGVhM2ExMDU3OWI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAxVDEyNDA0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlZDg0YTdiMDcxZDdiN2RjNWJmODEwMzk5MDI3YWE2MWYyNmU5OTU1NTgyMmE4MTIwMGNmZTUxY2UwOTk2ZTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tL4qx74xugCIK4U1EJAWcrOhHacTvyEIoowiJaAiQSA)

the other helmets are not heavy

Reply:
you're right my bad!